test/image-diff_test.js

```diff
@@ -45,12 +45,51 @@ describe('image-diff', function () {
     });
   });
 
+  describe('diffing different images using lower threshold', function () {
+    runImageDiff({
+      actualImage: __dirname + '/test-files/checkerboard.png',
+      expectedImage: __dirname + '/test-files/white.png',
+      diffImage: __dirname + '/actual-files/different.png',
+      threshold: 0.7
+    });
+    imageUtils.loadActual('different.png');
+    imageUtils.loadExpected('different.png');
+
+    it('asserts images are different', function () {
+      assert.strictEqual(this.imagesAreSame, false);
+    });
+
+    it('writes a highlighted image diff to disk', function () {
+      assert.deepEqual(this.actualPixels, this.expectedPixels);
+    });
+  });
+
+  describe('diffing different images using higher threshold', function () {
+    runImageDiff({
+      actualImage: __dirname + '/test-files/checkerboard.png',
+      expectedImage: __dirname + '/test-files/white.png',
+      diffImage: __dirname + '/actual-files/different2.png',
```

Comment:
As with the other comment, can we rename this to be something like `threshold-under.png`?